products/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+
+
+class MainCategory(models.Model):
+    name = models.CharField(max_length=50)
+
+    class Meta:
+        db_table = 'maincategories'
```

Comment:
table 명은 snake_case로 만들어주세요~
```suggestion
        db_table = 'main_categories'
```